packages/edit-site/src/components/sidebar-edit-mode/page-panels/edit-template.js

```diff
@@ -83,6 +93,20 @@ export default function EditTemplate() {
 							<SwapTemplateButton onClick={ onClose } />
 						</MenuGroup>
 						<ResetDefaultTemplate onClick={ onClose } />
+						<MenuItem
+							onClick={ () => {
+								setPageContentFocusType(
+									pageContentFocusType === 'disableTemplate'
+										? 'hideTemplate'
+										: 'disableTemplate'
+								);
+								onClose();
```

Comment:
Do folks think we need any `info` help text for this menu item or are the labels adequate?